source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -406,8 +411,8 @@ GLSSharpNavierStokesSolver<dim>::force_on_ib()
                           // Check if that dof already have been used to
                           // extrapolate the fluid stress tensor on the IB
                           // surface.
-                          if (force_eval_done[local_face_dof_indices[i]]
-                                .first == false)
+                          if (force_eval.find(local_face_dof_indices[i]) ==
+                              force_eval.end())
```

Comment:
Could you check if this is slower than the original line?

Reply:
if anything it's faster! (theoretically)